go.mod

```diff
@@ -4,6 +4,7 @@ go 1.14
 
 require (
 	github.com/ghodss/yaml v1.0.0
+	github.com/go-kit/kit v0.10.0 // indirect
```

Comment:
I couldn't find a reason why this got here & the sum file too. Any ideas?

Reply:
Ah yes, this must have been added when I was searching for `StatusCoder` and accidentally auto imported go-kit. Will clean up